src/cli/commands/__tests__/lint.test.ts

```diff
@@ -140,12 +140,7 @@ describe('lint', () => {
         .command(['lint', validOas3SpecPath, '-r', invalidRulesetPath, '-r', validRulesetPath])
         .exit(2)
         .it('given one is valid other is not, outputs "invalid ruleset" error', ctx => {
-          expect(ctx.stdout).toContain(`/rules/rule-without-given-nor-them 	 should have required property 'given'`);
```

Comment:
This is the only breaking change introduced in this PR - no more detailed rulesets validation info.
Didn't really mean to change it, but somehow did it, mostly due to the change around `validateRuleset` function.
I can revert it if needed.

Reply:
When you say detailed rulesets validation info, what do you mean? We no longer check to see if rulesets are valid? Or if they are invalid, we no longer give very good information about the problem? 

We should probably bring that back?

Reply:
> When you say detailed rulesets validation info, what do you mean? We no longer check to see if rulesets are valid? Or if they are invalid, we no longer give very good information about the problem?

We do check to see whether rules are invalid, but don't report any detailed errors anymore.
Before Spectral errored with very specific information about the error(s) in a provided ruleset. Right now, it just says ruleset is wrong yet doesn't tell you anything more.
I'll bring this back.